node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -219,6 +234,7 @@ object Configuration {
 
 final case class Configuration(
     underlying: Config,
+    unknownConfigurationKeys: Set[String],
```

Comment:
Wouldn't `unknownKeys` be enough given the fact that this is part of the `Configuration`?

Reply:
true

Reply:
Well, now I think `unknownKeys` is not sufficient. I'll rename it to `unknownConfigKeys` since the keys from in the `underlying: Config`.